mousestyles/tests/test_distribution.py

```diff
@@ -0,0 +1,10 @@
+# coding: utf-8
+from mousestyles.distribution import (powerlaw_pdf, exp_pdf)
+
+
+def test_powerlaw_pdf():
+    assert (powerlaw_pdf(2, 2) == 0.25)
+
+
+def test_exp_pdf():
+    assert (exp_pdf(2, 1) == 0.36787944117144233)
```

Comment:
I am not sure whether this test function will work for exponential pdf. How about we calculate the difference and see if difference is small enough. For example, we can test on 

abs(exp_pdf(2, 1) - 0.367879 <= 1e-7)